src/components/SpaceCreateWarnings.vue

```diff
@@ -37,12 +41,31 @@ const strategySymbolsString = computed(() => {
 
   return `(${symbols.join(', ')})`;
 });
+
+const isAdmin = computed(() => {
+  const admins = (props.space.admins || []).map(admin => admin.toLowerCase());
+  return admins.includes(web3Account.value?.toLowerCase());
+});
 </script>
 
 <template>
   <div class="mb-4 space-y-2">
+    <BaseMessageBlock
+      v-if="space.hibernated"
+      level="warning"
+      is-responsive
+    >
+      {{$t('create.errorSpaceHibernated')}}
+      <BaseLink link="https://docs.snapshot.org/">
```

Comment:
Link needs to be updated, there is already a page on the docs for hibernation

Reply:
Can you share it ?